panda_leader_cartesian_impedance_controller:
  type: panda_teleoperation/LeaderCartesianImpedanceController
  # type: thorvald_panda_controllers/LeaderCartesianImpedanceController
  arm_id: panda_leader
  joint_names:
      - panda_leader_joint1
      - panda_leader_joint2
      - panda_leader_joint3
      - panda_leader_joint4 
      - panda_leader_joint5
      - panda_leader_joint6
      - panda_leader_joint7

panda_leader_state_controller:
  type: franka_control/FrankaStateController
  publish_rate: 30  # [Hz]
  joint_names:
    - panda_leader_joint1
    - panda_leader_joint2
    - panda_leader_joint3
    - panda_leader_joint4
    - panda_leader_joint5
    - panda_leader_joint6
    - panda_leader_joint7
  arm_id: panda_leader
